web/src/components/organisms/Settings/Members/hooks.ts

```diff
@@ -88,9 +93,11 @@ export default ({ workspaceId }: Props) => {
             }
           : undefined,
       )
-      .filter((user): user is Member => !!user)
+      .filter(
+        (user): user is Member => !!user && user.user.name.toLowerCase().includes(searchTerm ?? ""),
```

Comment:
```suggestion
        (user): user is Member => !!user && user.user.name.toLowerCase().includes(searchTerm.toLowerCase() ?? ""),
```

Reply:
Do we want to make the search case-sensitive?

Reply:
@KaWaite , no it should be insensitive.
Fixed.